api/src/test/java/org/openmrs/module/fhir2/FhirTestConstants.java

```diff
@@ -14,4 +14,7 @@
 	public static final String LOINC_SYSTEM_URL = "http://loinc.org";
 	
 	public static final String CIEL_SYSTEM_URN = "urn:oid:2.16.840.1.113883.3.7201";
+	
+	public static final String PATIENT = "Patient";
```

Comment:
Just use the value from FhirConstants. The purpose of FhirTestConstants is for some constants needed across tests that aren't constants in the application, e.g., the `LOINC_SYSTEM_URL` or `CIEL_SYSTEM_URN` are actually determined at runtime from the database, but for testing purposes, we can use fixed values.

Reply:
👍🏽